src/sink.h

```diff
@@ -240,19 +242,19 @@ class Sink
                       "tooltip": "lower bound of the random frequency", \
                       "uilabel": "Random Frequency Lower Bound", \
                       "uitype": "range", \
-                      "range": [1, 1e299], \
+                      "range": [1, CY_LARGE_INT], \
                       "doc": "When a random distribution is used to determine the " \
                              "frequency of the request, this is the lower bound. Default 1"}
   int random_frequency_min;
 
   // random frequency upper bound
-  #pragma cyclus var {"default": 1e299, \
+  #pragma cyclus var {"default": CY_LARGE_INT, \
                       "tooltip": "upper bound of the random frequency", \
                       "uilabel": "Random Frequency Upper Bound", \
                       "uitype": "range", \
-                      "range": [1, 1e299], \
+                      "range": [1, CY_LARGE_INT], \
                       "doc": "When a random distribution is used to determine the " \
-                             "frequency of the request, this is the upper bound. Default 1e299"}
+                             "frequency of the request, this is the upper bound. Default CY_LARGE_INT (1e299)"}
```

Comment:
Can we get the actual value of the variable to appear here rather than hardcoding 1e299 in the string?  Would this work?
```suggestion
                             "frequency of the request, this is the upper bound. Default CY_LARGE_INT (" + str(CY_LARGE_INT) + ")"}
```
I think this is ultimately processed in python, so not sure how the value of this metadata is passed along.

Reply:
Good catch, this is now replaced at build time